Zilon.Core/Zilon.Core/Persons/MonsterPerson.cs

```diff
@@ -3,69 +3,66 @@
 
 using JetBrains.Annotations;
 
-using Zilon.Core.Props;
+using Zilon.Core.PersonModules;
 using Zilon.Core.Schemes;
 
 namespace Zilon.Core.Persons
 {
     /// <summary>
     /// Персонаж для монстров в секторе.
     /// </summary>
-    public class MonsterPerson : IPerson
+    public class MonsterPerson : PersonBase
     {
-        public int Id { get; set; }
-        public int Hp { get; }
-        public IEquipmentCarrier EquipmentCarrier => null;
-
-        public ITacticalActCarrier TacticalActCarrier { get; }
-
-        public IEvolutionData EvolutionData => throw new NotSupportedException("Для монстров не поддерживается развитие");
-
-        public ICombatStats CombatStats { get; }
+        /// <inheritdoc/>
+        public override int Id { get; set; }
 
-        public IPropStore Inventory => throw new NotSupportedException("Для монстров не поддерживается инвентарь.");
-
-        public ISurvivalData Survival { get; }
-
-        public EffectCollection Effects { get; }
+        /// <inheritdoc/>
+        public int Hp { get; }
 
+        /// <inheritdoc/>
         public IMonsterScheme Scheme { get; }
 
-        public PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
-        public bool HasInventory { get => false; }
-        public IDiseaseData DiseaseData { get; }
+        /// <inheritdoc/>
+        public override PhysicalSize PhysicalSize { get => PhysicalSize.Size1; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5453268)